astropy/visualization/scripts/fits2bitmap.py

```diff
@@ -234,14 +253,22 @@ def main(args=None):
     )
     args = parser.parse_args(args)
 
+    if args.min_cut is not None:
```

Comment:
Do you have to check for the case where both `--min_cut` and `--vmin` are given?

Reply:
tests added, thanks